src/DS2431.h

```diff
@@ -0,0 +1,28 @@
+// 0x2D  1Kb 1-Wire EEPROM
+// tested on buspirate and two different real 1-wire masters
```

Comment:
would be nice to hear about the masters. one seems to be the ds9490r. and the other? i plan on putting a compatibility-chart into the main readme.md